Assets/GothicVR/Scripts/Manager/GvrSceneManager.cs

```diff
@@ -53,18 +54,30 @@ public async Task LoadStartupScenes()
                 else
                     await LoadMainMenu();
 
-            if (FeatureFlags.I.CreateOcNpcs)
-                PxVm.CallFunction(GameData.I.VmGothicPtr, "STARTUP_SUB_OLDCAMP");
-
-                if (FeatureFlags.I.CreateDebugIdleAnimations)
-                    NpcCreator.I.DebugAddIdleAnimationToAllNpc();
+                debugFreshlyDoneLoading = true;
             }
             catch (Exception e)
             {
                 Debug.LogError(e);
             }
         }
 
+        // Outsourced after async Task LoadStartupScenes() as async makes Debugging way harder
+        // (Breakpoints won't be catched during exceptions)
+        private void Update()
+        {
+            if (!debugFreshlyDoneLoading)
+                return;
+            else
+                debugFreshlyDoneLoading = false;
+            
+            if (FeatureFlags.I.CreateOcNpcs)
+                PxVm.CallFunction(GameData.I.VmGothicPtr, "STARTUP_SUB_OLDCAMP");
+
+            if (FeatureFlags.I.CreateDebugIdleAnimations)
+                NpcCreator.I.DebugAddIdleAnimationToAllNpc();
+        }
+
```

Comment:
NPCs are loaded only if we directly load into the world, but not if we go through the main menu, is this intended?